homeassistant/components/websocket_api/http.py

```diff
@@ -161,6 +166,10 @@ def handle_hass_stop(event):
                     break
 
                 self._logger.debug("Received %s", msg)
+                self.hass.bus.async_fire(
+                    EVENT_WEBSOCKET_REQUEST,
```

Comment:
Please remove this:

- It is going to fire A LOT OF EVENTS
- It is not being used

Reply:
Yes... I agree this will fire a lot of events. 

We do have the same thing happening with the google assistant component, hence I put it there. Will remove it.